docs/intro/tutorial.rst

```diff
@@ -41,32 +41,30 @@ directory where you'd like to store your code and run::
 This will create a ``tutorial`` directory with the following contents::
 
     tutorial/
-        scrapy.cfg
-        tutorial/
+        scrapy.cfg            # deploy configuration file
+
+        tutorial/             # project's Python module, you'll import your code from here
             __init__.py
-            items.py
-            pipelines.py
-            settings.py
-            spiders/
+
+            items.py          # project items file
+
+            pipelines.py      # project pipelines file
+
+            settings.py       # project settings file
+
+            spiders/          # a directory where you'll later put your spiders
                 __init__.py
                 ...
```

Comment:
So, with the risk of overthinking this... :P
The main change here was to avoid the indirection (before it was showing the structure, and detailing the type of content below).

Yes, it is a bit more sparse now, but this is nicer for a reader trying to understand the structure, because 1) she can see the descriptions directly in the structure tree, and 2) she can scan quickly the vertically-aligned comments to know the kinds of files there are.

The only thing I'm not so sure about here is the vertical alignment.